cmd/sponged/handlers/data.go

```diff
@@ -0,0 +1,49 @@
+// Package handlers contains the handler logic for processing requests.
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/ardanlabs/kit/db"
+	"github.com/ardanlabs/kit/web/app"
+	"github.com/coralproject/shelf/internal/sponge/item"
+)
+
+const defaultVersion = 1
```

Comment:
Is this consistent with everything else we are doing. If not let's remove it or make it consistent everywhere. Not sure what this represents?